### PR TITLE
Python (Black) and XML Lint PR Action

### PR DESCRIPTION
**Pull request by cpqoc**
## Task

https://qocinnovations.odoo.com/web#id=8896&cids=1&model=project.task&view_type=form

## Description

PR Action to automate review: Black Linting

### CODE REVIEW

File: .github/DELETE_ME_BEFORE_MERGE_TO_MAIN/DELETE_ME.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+print("Test")
+data = {
+    'a': 123,
+    'b': 456,
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad': 123 + 4567 + 976843576985432798652492552 + 98734589762345978
+}
```
Comment (TFqoc) [inline .github/DELETE_ME_BEFORE_MERGE_TO_MAIN/DELETE_ME.py:6]:
```suggestion
```
Take a look at the filename for explanation here.

**Review by TFqoc — CHANGES_REQUESTED** [review]
One change, then we can give this a spin!

File: .github/DELETE_ME_BEFORE_MERGE_TO_MAIN/DELETE_ME.py (hunk 1 above)
Comment (cpqoc) [reply .github/DELETE_ME_BEFORE_MERGE_TO_MAIN/DELETE_ME.py]: Removed in squash

**Comment by cpqoc** [comment]
@TFqoc Sorry about all the emails lol

**Comment by cpqoc** [comment]
Not sure why the "squash" didn't trigger the checks (Soft reset HEAD locally, commit all changes in one commit, push).

Probably don't want do that in the future lol

**Comment by cpqoc** [comment]
> Not sure why the "squash" didn't trigger the checks (Soft reset HEAD locally, commit all changes in one commit, push).
> 
> Probably don't want do that in the future lol

Ah, it's because I removed the testing files. Duh, of course it's not firing, that's a feature lol

**Pull request merged**